gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -370,6 +371,30 @@ public static boolean intersectFrustumBounds (Frustum frustum, BoundingBox bound
 		return frustumIsInsideBounds;
 	}
 
+	/** Returns whether the given {@link Frustum} intersects a {@link OrientedBoundingBox}.
+	 * @param frustum The frustum
+	 * @param obb The oriented bounding box
+	 * @return Whether the frustum intersects the oriented bounding box */
+	public static boolean intersectFrustumBounds (Frustum frustum, OrientedBoundingBox obb) {
```

Comment:
I am not sure what I am doing but the references makes sense I just based the code on the following links:

https://www.gamedev.net/forums/topic/539116-optimized-obb-frustum/4479551/
https://gist.github.com/Kinwailo/d9a07f98d8511206182e50acda4fbc9b

Reply:
@mgsx-dev there are other cases failing too, basically the part that checks if the frustum points are inside the BoundingBox are not working properly.

I am trying to find a cleaner way to implement this:
```
int out;
    out=0; for( int i=0; i<8; i++ ) out += ((fru.mPoints[i].x > box.mMaxX)?1:0); if( out==8 ) return false;
    out=0; for( int i=0; i<8; i++ ) out += ((fru.mPoints[i].x < box.mMinX)?1:0); if( out==8 ) return false;
    out=0; for( int i=0; i<8; i++ ) out += ((fru.mPoints[i].y > box.mMaxY)?1:0); if( out==8 ) return false;
    out=0; for( int i=0; i<8; i++ ) out += ((fru.mPoints[i].y < box.mMinY)?1:0); if( out==8 ) return false;
    out=0; for( int i=0; i<8; i++ ) out += ((fru.mPoints[i].z > box.mMaxZ)?1:0); if( out==8 ) return false;
    out=0; for( int i=0; i<8; i++ ) out += ((fru.mPoints[i].z < box.mMinZ)?1:0); if( out==8 ) return false;
```
https://iquilezles.org/www/articles/frustumcorrect/frustumcorrect.htm

Reply:
Note to myself, one idea is to calculate Vector3 min, max; and check based on box bounds.

Reply:
@mgsx-dev I added the ugly code because it is slightly more efficient than what I had in mind. The example you gave is now printing `true, true` I think that's the expected outcome right?

I added a new test sample that draws a Frustum and boxes so it's easier to debug. The collisions are not working perfectly for AABB but I don't want to fix them here in this PR.

What do you think about the PR, is it good to go?

Reply:
@mgsx-dev nvm, I am rolling back the code, this code cover some edge cases but there are others remaining.